manifests/base/redis/argocd-redis-deployment.yaml

```diff
@@ -30,9 +30,6 @@ spec:
         - ""
         - "--appendonly"
         - "no"
-        env:
-        - name: ARGOCD_REDIS_SERVICE
-          value: ""
```

Comment:
I don't think this does anything. We're using the upstream redis image, which has no reason to know anything about an Argo CD env var.